text/3817-promote-aarch64-pc-windows-msvc-to-tier-1.md

```diff
@@ -0,0 +1,127 @@
+- Feature Name: promote-aarch64-pc-windows-msvc-to-tier-1
+- Start Date: 2025-05-22
+- RFC PR: [rust-lang/rfcs#3817](https://github.com/rust-lang/rfcs/pull/3817)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Promote aarch64-pc-windows-msvc to Tier 1 with Host Tools.
+
+# Motivation
+[motivation]: #motivation
+
+About [30% of Rust users use Windows][survey-2024], while the majority of these developers and their
+customers are using x64 hardware, the usage of Arm64 Windows has been growing since it was first
+made available in Windows 10, and has been accelerating, especially with the availability of the
+SnapDragon X processors.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+No changes required: Rust tooling for Arm64 Windows has been available for a while now so this
+doesn't affect the end user experience.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Tier 1 targets must adhere to the [Tier 1 Target Policy][tier-1-policy]. Going through these
+requirements point-by-point:
+
+> Tier 1 targets must have substantial, widespread interest within the developer community, and must
+> serve the ongoing needs of multiple production users of Rust across multiple organizations or
+> projects. These requirements are subjective, and determined by consensus of the approving teams.
+
+As mentioned above, Windows users comprise a substatial propotion of Rust developers, and Arm64
+hardware is increasingly being used by them and their customers.
+
+> The target maintainer team must include at least 3 developers.
+
+There is an existing [Windows team for Rust][windows-team] and, within Microsoft, Arm64 Windows is
```

Comment:
Are there at least 3 people in that group that are willing to be active maintainers for this particular target? It seems like we should identify who those people will be.

The windows marker team is mainly just a [ping group](https://forge.rust-lang.org/triagebot/pinging.html) and not necessarily maintainers (anyone can be added to it).

Reply:
Yeah, Tier 1 target policy requires

> The target maintainer team must include at least 3 developers.

The ping group is advisory at best, it is not even specific to the other Tier 1 `x86_64-pc-windows-msvc` or `x86_64-pc-windows-gnu` targets.

On the compiler team, we lack expertise bandwidth on the Tier 1 Windows targets already. Furthermore, to the best of my knowledge, very few (if any) compiler team members have access to a `aarch64-pc-windows-msvc` host, be it natively or remotely. This makes even "routine" tasks like triaging or bisection or minimizations very difficult, so for a Tier 1 target, we would have to rely on its target maintainers perhaps more than is the case for other Tier 1 targets.

Reply:
Fair enough: let me talk to a few folks internally, I'd like to get an official `*-pc-windows-msvc` maintainer list setup for the various Windows targets.

Reply:
Thanks! After that PR is merged, you should mention that here.